home-assistant-plugin/custom_components/xknx/__init__.py

```diff
@@ -140,15 +140,21 @@
 SERVICE_XKNX_SEND_SCHEMA = vol.Any(
     vol.Schema(
         {
-            vol.Required(SERVICE_XKNX_ATTR_ADDRESS): cv.string,
+            vol.Required(SERVICE_XKNX_ATTR_ADDRESS): vol.All(
+                cv.ensure_list,
+                [cv.string],
```

Comment:
I guess this makes sure the list can contain one element of type string? (backwards compatibility)

Reply:
Yep 👍